books/political-science/bake

```diff
@@ -34,8 +34,7 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :political_science) do |doc|
       metadata_source: metadata,
       container_key: 'suggested-readings',
       uuid_key: '.suggested-readings',
-      section_selector: 'section.suggested-readings',
-      include_intro_page: true
+      section_selector: 'section.suggested-readings'
     ) do |section|
       RemoveSectionTitle.v1(section: section)
     end
```

Comment:
I think we should update also these parts: 
```
BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata, klass: 'review-questions')
```
to use `MoveCustomSectionToEocContainer`. And Political Science should use injected exercises now I guess :thinking: